package.json

```diff
@@ -27,6 +27,9 @@
     "type": "git",
     "url": "https://github.com/react-dropzone/file-selector.git"
   },
+  "publishConfig": {
```

Comment:
What will this publish or where can one see the side effect of this?

Reply:
This is used to indicated that this package should be publishable when running `npm publish`, it is similar to `private: false`. This is enforced for scoped packages and for our case providing a provenance statement.